maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/main/java/org/apache/maven/scm/provider/svn/svnexe/command/status/SvnStatusConsumer.java

```diff
@@ -169,20 +169,9 @@ else if ( statusString.equals( "I" ) )
             //Parse the second column
             statusString = line.substring( 1, 1 );
 
-            if ( statusString.equals( "M" ) )
-            {
-                status = ScmFileStatus.MODIFIED;
-            }
-            else if ( statusString.equals( "C" ) )
-            {
-                status = ScmFileStatus.CONFLICT;
-            }
-            else
-            {
-                //The line isn't a status line, ie something like 'Performing status on external item at...'
-                //or a status defined in next columns
-                return;
-            }
+            //The line isn't a status line, ie something like 'Performing status on external item at...'
+            //or a status defined in next columns
+            return;
```

Comment:
Please explain

Reply:
"M" and ¨C" are cover in previous checks

`else if ( statusString.equals( "M" ) || statusString.equals( "R" ) || statusString.equals( "~" ) )`
and
`  else if ( statusString.equals( "C" ) )
        {
            status = ScmFileStatus.CONFLICT;
        }`